R/inline.R.in

```diff
@@ -19,11 +19,11 @@ inlineCxxPlugin <- function(...) {
     plugin <-
         Rcpp::Rcpp.plugin.maker(
                    include.before = "#include <RcppArmadillo.h>",
-                   libs           = "$(SHLIB_OPENMP_CFLAGS) $(LAPACK_LIBS) $(BLAS_LIBS) $(FLIBS)",
+                   libs           = "@HAVE_OPENMP_R_CFLAG@ $(LAPACK_LIBS) $(BLAS_LIBS) $(FLIBS)",
```

Comment:
Making this a package configure-time check doesn't feel right to me, IMHO. This implies that a package built on CRAN might detect OpenMP support, which implies that assumption will remain when the Rcpp binary is later served to the user. However, that assumption will fail if the user has not installed the CRAN LLVM toolchain.

I would prefer that we just checked this at runtime, rather than configure time.

Reply:
Yes, but how do you suggest we go about it?

I started down the road of 'grep'ing CXXFLAGS and looking for `-fopenmp`.  As you know that fails on macOS.

Reply:
No matter what, we do need a compile time check to set the `src/Makevars` appropriately. This was evident with the clang-sans-openmp check Ripley is running. Regarding the build check, I do agree that would be beneficial; however, there _really_ isn't a good way to detect OpenMP on macOS reliably as support for OpenMP is not listed within `capabilities()`.

My only thought to that line of detection is the possibility of checking for the _directory_ existence of `/usr/local/clang4` on macOS. Other platforms have a tendency to respect setting -- or unsetting -- `SHLIB_OPENMP_*` where appropriate.